tests/test_callback.py

```diff
@@ -0,0 +1,37 @@
+import os
+import re
+import ansible_runner
+
+
+def run_playbook(tmpdir):
+    os.environ['ANSIBLE_STDOUT_CALLBACK'] = "foreman"
```

Comment:
if you use `theforeman.foreman.foreman` here, you don't need to make the symlink for the `foreman.py` file, which I think is nicer.